lib/rawHttpRequest/rawHttpRequestSchema.js

```diff
@@ -0,0 +1,201 @@
+module.exports = {
+
+	title: 'Raw HTTP Request (Advanced)',
+
+	globals: false,
+
+	input: {
+
+		method: {
+			type: 'string',
+			description: 'The HTTP verb to perform the request with',
+			enum: [
+				'GET',
+				'POST',
+				'PATCH',
+				'PUT',
+				'DELETE',
+				'HEAD',
+				'OPTIONS'
+			],
+		},
+
+		url: {
+			description: 'The URL to make the request with.',
+			oneOf: [
+
+				{
+					title: 'Endpoint',
+					type: 'object',
+					properties: {
+
+						endpoint: {
+							type: 'string',
+							description: 'The endpoint to call in relation to the base URL used by the connector.',
+							required: true
+						}
+
+					}
+				},
+
+				{
+					title: 'Full URL',
+					type: 'object',
+					properties: {
+
+						full_url: {
+							type: 'string',
+							description: 'The full URL to make the request against.',
+							required: true
+						}
+
+					}
+				}
+
+			],
+			required: true
+		},
+
+		headers: {
+			type: 'object',
+			description: 'The headers required to perform the request with.',
+			additionalProperties: true
+		},
+
+		queries: {
+			title: 'Query Parameters',
+			type: 'object',
+			description: 'The URL query parameters to perform the request with.',
```

Comment:
what format will this send arrays? maybe the user should be aware of this

Reply:
The behaviour should be the same as the HTTP client.

That said, I just double checked and realised this `queries ` schema is not the same the HTTP client - will update.